docs/graphql/core/schema/remote-relationships/remote-schema-relationship-permissions.rst

```diff
@@ -0,0 +1,27 @@
+.. meta::
+   :description: Remote schema relationship permissions
+   :keywords: authorization, docs, remote schema relationship, remote schema relationship permissions
+
+.. _remote_schema_relationship_permissions:
+
+Remote schema relationship permissions
```

Comment:
I'm not sure if this should be a separate section. How about making it a top level section [here](https://hasura.io/docs/1.0/graphql/core/schema/remote-relationships/remote-schema-relationships.html)? On the same level as `Introduction` but in the end?

Reply:
Sure, I'll do that!